gui/src/components/Taipy/Slider.tsx

```diff
@@ -68,49 +69,53 @@ const Slider = (props: SliderProps) => {
     const max = lovList.length ? lovList.length - 1 : props.max;
     const horizontalOrientation = props.orientation ? props.orientation.charAt(0).toLowerCase() !== "v" : true;
 
+    // Converts the slider value (number or array of numbers) to a proper backend value
+    // depending on whether we're dealing with a lov or not
+    const convertValue = useCallback(
+        (v: number | number[]): number | number[] | string | string[] =>
+            Array.isArray(v)
+            ? (lovList.length
+                ? v.map((i) => lovList.length > (i as number) ? lovList[i as number].id : lovList[0].id)
+                : v)
+                : (lovList.length && lovList.length > (v as number) ? lovList[v as number].id : v),
+        [lovList])
+
     const handleRange = useCallback(
         (e: Event, val: number | number[]) => {
-            setValue(val as number);
+            setValue(val);
             if (update) {
-                lastVal.current = lovList.length && lovList.length > (val as number) ? lovList[val as number].id : val as number;
```

Comment:
is lastVal (useRef) still usefull when we have the action in the scope ?

Reply:
My bad